nss-day-cohort-46/rare-client-groovy-goblins
# Feat: create a tag

# Description

Added provider and route to render a tag-form. Users can now add tags. 

Fixes: Create A Tag

## Type of change


# Testing Instructions

```
git fetch --all
git checkout dr-createATag
npm start
```
checkout to SERVER branch as well
go to: http://localhost:3000/tags/create

Type in the text field then submit.

To see the new tag, from your .sql file run: 
```SELECT * FROM Tags;```

# Checklist:
